package.json

```diff
@@ -78,6 +78,12 @@
       }
     }
   },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-push": "yarn prettier:fix"
```

Comment:
do we really want prettier:fix runned on pre-push? I would have it also for commit

Reply:
Well `commit-msg` is validated after `pre-commit` making it little painfull to wait for prettier to run before commit is validated, it still takes couple of seconds which is totally ok for push, not so sure about commit though